src/types.rs

```diff
@@ -66,6 +66,36 @@ impl fmt::Debug for dyn Iden {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum Identity {
```

Comment:
@billy1624 can you explain, why you move this enum to SeaQuery? I cannot find place, where we can use it in SeaQuery

Reply:
Hey @ikrivosheev, it's true that `Identity` and `IntoIdentity` aren't used in any part of SeaQuery at the moment. It originally reside in SeaORM because we replace SeaORM's `IdenStatic` with `sea_query::IdenStatic`, this make it impossible to define `impl<T: IdenStatic> IntoIdentity for T` and hence moving both into SeaQuery.

Btw, `IdenList` trait can be replaced by `IntoIdentity` in the future, i.e. methods that originally take `IdenList` parameter can now take `IntoIdentity` instead.